pkg/keys/keys.go

```diff
@@ -517,12 +517,13 @@ func SpanAddr(span roachpb.Span) (roachpb.RSpan, error) {
 // RangeMetaKey returns a range metadata (meta1, meta2) indexing key for the
 // given key.
 //
-// - For RKeyMin, KeyMin is returned.
-// - For a meta1 key, KeyMin is returned.
+// - For RKeyMin, or RKeyMin.Next(), RKeyMin is returned.
```

Comment:
mind being more precise here, in that we return `RKeyMin` for anything smaller than the meta1 prefix. Perhaps also which of these cases we actually hit (or expect to hit at least).